index.qmd

```diff
@@ -1,10 +1,25 @@
 ---
 title: "The Journal of Visualization and Interaction"
+bibliography: biblio.bib
 subtitle: A [diamond open-access](https://en.wikipedia.org/wiki/Open_access#Diamond/platinum_OA) journal for the [InfoVis](http://ieeevis.org/) and [HCI](https://chi.acm.org/) communities.
 freeze: true
+format:
+  html:
+    filters:
+      - empty.lua
+    myrefs: ""
+  pdf:
+    link-citations: true
+    number-sections: true
+    myrefs: "References"
+    author:
+      - "[Lonni Besançon](https://lonnibesancon.me/), Linköping University"
+      - "[Florian Echtler](https://floe.butterbrot.org/), Aalborg University"
+      - "[Matthew Kay](https://www.mjskay.com/), Northwestern University"
+      - "[Chat Wacharamanotham](https://chatchavan.github.io), Swansea University"
 ---
 
-## What is JoVI?
+# What is JoVI?
```

Comment:
Why change this to a level-1 heading?

Reply:
If you render it as PDF, you get a numbering of 0.1, 0.2 etc. otherwise.